### PR TITLE
chore(sankey): polish sankey docs

### PR DESCRIPTION
**Pull request by ntacheva**
Closes https://github.com/telerik/blazor/issues/9270

### CODE REVIEW

File: components/sankey/elements/links.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ To customize the nodes, declare the `<SankeyLinks>` tag as a direct child of `<T
 
 | Parameter | Type and Default&nbsp;Value | Description |
 | --------- | ---- | ----------- |
-| `ColorType` | `SankeyLinksColorType` enum <br /> (`Static`) | The origin of the link color: <ul><li>`Static` - the link color is based on the `Color` property;</li><li>`Source` - the link color is based on the source node color; </li><li>`Target` - the link color is based on the target node color; </li></ul> |
+| `ColorType` | [`SankeyLinksColorType` enum](/blazor-ui/api/telerik.blazor.sankeylinkscolortype) <br /> (`SankeyLinksColorType.Static`) | The origin of the link color: could be based on the `Color` property, on the source node color or on the target node color. |
```
Comment (ikoevska) [inline components/sankey/elements/links.md:24]: In a list of three or more items, the serial comma is a good practice. ([source](https://docs.telerik.com/style-guide/punctuation#series))

```suggestion
| `ColorType` | [`SankeyLinksColorType` enum](/blazor-ui/api/telerik.blazor.sankeylinkscolortype) <br /> (`SankeyLinksColorType.Static`) | The origin of the link color: could be based on the `Color` property, on the source node color, or on the target node color. |
```

File: components/sankey/elements/tooltip.md [hunk 2]
```diff
@@ -24,36 +24,26 @@ The `LinkTemplate` controls the content of the Tooltip that will appear when the
 
 | Property | Type | Description |
 | ---------| ---- | ----------- |
-| `Source` | `SankeyDataNode` | The source of the hovered link. Provides details for the source node such as its label, opacity, color, width, offset, alignment and more.   |
-| `Target` | `SankeyDataNode` | The target of the hovered link. Provides details for the target node such as its label, opacity, color, width, offset, alignment and more.   | 
-| `Value` | `double?` | The hovered link value.  | 
+| `Source` | [`SankeyDataNode`](/blazor-ui/api/telerik.blazor.sankeydatanode) | The source of the hovered link. Provides details for the source node such as its label, opacity, color, width, offset, alignment and more.   |
```
Comment (ikoevska) [inline components/sankey/elements/tooltip.md:27]: A good candidate for the serial comma. Not marking any more instances.

```suggestion
| `Source` | [`SankeyDataNode`](/blazor-ui/api/telerik.blazor.sankeydatanode) | The source of the hovered link. Provides details for the source node such as its label, opacity, color, width, offset, alignment, and more.   |
```

File: components/sankey/data-bind.md [hunk 3]
```diff
@@ -49,10 +49,10 @@ The `SankeyDataNode` object contains all the information for the node and its la
 | `Id` | `object` | The ID of the node. The ID is used to connect the nodes with the links. Required. |
 | `Label` | [`SankeyDataNodeLabel`](/blazor-ui/api/Telerik.Blazor.Components.SankeyDataNodeLabel) | Contains all the information for the node label - text, alignment, color, border and more. |
 | `Offset` | [`SankeyDataNodeOffset`](/blazor-ui/api/Telerik.Blazor.Components.SankeyDataNodeOffset)| The left and top offset of the node from the `<div class="k-sankey">` container. |
-| `Opacity` | `double?` | The opacity of the node. |
+| `Opacity` | `double?` <br/> (`1`) | The opacity of the node. |
 | `Padding` | `double?` | The minimum vertical space between two nodes. |
-| `Width` | `double?` | The width of the node. |
-| `Align` | `SankeyNodesAlign?` enum <br/> (`SankeyNodesAlign.Stretch`) | The alignment of the node. Supports the following values - `Stretch`, `Left`, and `Right`.|
+| `Width` | `double?` <br/> (`24`)| The width of the node. |
+| `Align` | [`SankeyNodesAlign?` enum](/blazor-ui/api/telerik.blazor.sankeynodesalign) <br/> (`SankeyNodesAlign.Stretch`) | The alignment of the node.|
```
Comment (dimodi) [inline components/sankey/data-bind.md:55]: The enum type is known, so you can skip it to shorten the value (and the column width).

```suggestion
| `Align` | [`SankeyNodesAlign?` enum](/blazor-ui/api/telerik.blazor.sankeynodesalign) <br/> (`Stretch`) | The alignment of the node.|
```

File: components/sankey/data-bind.md [hunk 4]
```diff
@@ -30,9 +30,9 @@ The `SankeyDataLink` object contains all the information for the link. It expose
 | Property | Type and Default&nbsp;Value | Description |
 | --------- | ---- | ----------- |
 | `ColorType` | `SankeyLinksColorType` enum <br/> (`SankeyLinksColorType.Static`) | The color type of the link. Provides the following values <ul><li>`Static` the link color is set based on the `Color` property;</li><li>`Source` - the link color is set based on the source node color; </li><li>`Target` - the link color is set based on the target node color; </li></ul> |
```
Comment (dimodi) [inline components/sankey/data-bind.md:32]:
```suggestion
| `ColorType` | `SankeyLinksColorType` enum <br/> (`Static`) | The color type of the link. Provides the following values <ul><li>`Static` the link color is set based on the `Color` property;</li><li>`Source` - the link color is set based on the source node color; </li><li>`Target` - the link color is set based on the target node color; </li></ul> |
```

File: components/sankey/elements/labels.md [hunk 5]
```diff
@@ -21,10 +21,10 @@ To customize the labels, declare a `<SankeyLabels>` tag as a direct child of `<T
 
 | Parameter | Type and Default&nbsp;Value | Description |
 | --------- | ---- | ----------- |
-| `Align` | `string` | The alignment of the labels. |
-| `Color` | `string` | The color of the labels. |
-| `Font` | `string` | The font of the labels. |
-| `Position` | `string` <br/> (`inside`) | The position of the labels. The supported values are: <ul><li>`inside` - the label is positioned after the node, except for the nodes at the end of the Sankey, that are placed before the node;</li><li>`before` - the label is positioned before the node; </li><li>`after` - the label is positioned after the node; </li></ul> |
+| `Align` | [`SankeyLabelsAlign` enum](/blazor-ui/api/telerik.blazor.sankeylabelsalign) <br/> (`SankeyLabelsAlign.Left`) | The alignment of the labels. |
```
Comment (dimodi) [inline components/sankey/elements/labels.md:24]:
```suggestion
| `Align` | [`SankeyLabelsAlign` enum](/blazor-ui/api/telerik.blazor.sankeylabelsalign) <br/> (`Left`) | The alignment of the labels. |
```

File: components/sankey/elements/labels.md [hunk 6]
```diff
@@ -21,10 +21,10 @@ To customize the labels, declare a `<SankeyLabels>` tag as a direct child of `<T
...
+| `Color` | `string` <br/> (`rgb(66, 66, 66)`) | The color of the labels. |
+| `Font` | `string` <br/> (`14px Metric, Arial, Helvetica, sans-serif`)| The font of the labels. |
+| `Position` | [`SankeyLabelsPosition` enum](/blazor-ui/api/telerik.blazor.sankeylabelsposition) <br/> (`SankeyLabelsPosition.Inside`) | The position of the labels. |
```
Comment (dimodi) [inline components/sankey/elements/labels.md:27]:
```suggestion
| `Position` | [`SankeyLabelsPosition` enum](/blazor-ui/api/telerik.blazor.sankeylabelsposition) <br/> (`Inside`) | The position of the labels. |
```

File: components/sankey/elements/labels.md [hunk 7]
```diff
@@ -39,8 +39,8 @@ By design, the labels do not have border. You may add border by declaring the `<
 | Parameter | Type and Default&nbsp;Value | Description |
 | --------- | ---- | ----------- |
 | `Color` | `string` | The color of the border. |
-| `DashType` | `DashType` enum <br/> (`DashType.Solid`) | The type of the border. |
-| `Width` | `double?` | The width of the border. |
+| `DashType` | [`DashType` enum](/blazor-ui/api/telerik.blazor.dashtype) <br/> (`DashType.Solid`) | The type of the border. |
```
Comment (dimodi) [inline components/sankey/elements/labels.md:42]: Avoid explaining a term with the same word.

```suggestion
| `DashType` | [`DashType` enum](/blazor-ui/api/telerik.blazor.dashtype) <br/> (`Solid`) | The style of the border. |
```

File: components/sankey/elements/title.md [hunk 8]
```diff
@@ -26,13 +26,13 @@ The `<SankeyTitle>` tag exposes the following parameters for customization of th
 
 | Parameter | Type and Default&nbsp;Value | Description |
 | --------- | ---- | ----------- |
-| `Align` | `SankeyTitleAlign` enum | The alignment of the title. |
+| `Align` | [`SankeyTitleAlign` enum](/blazor-ui/api/telerik.blazor.sankeytitlealign) <br/> (`SankeyTitleAlign.Left`) | The alignment of the title. |
 | `Background` | `string`  | The background color of the title. |
-| `Color` | `string`  | The text color of the title. |
+| `Color` | `string` <br/> (`rgb(66, 66, 66)`)  | The text color of the title. |
```
Comment (dimodi) [inline components/sankey/elements/title.md:31]: Somewhere else I saw a default value in HEX format. Is the documentation inconsistent, or do we really set the same color in two different ways?

**Review by dimodi — APPROVED** [review]
Consider removing the enum types from the default value to save space:

(`EnumType.EnumValue`) => (`EnumValue`)

File: components/sankey/elements/title.md (hunk 8 above)
Comment (ntacheva) [reply components/sankey/elements/title.md:31]: You are probably referring to the [links color](https://docs.telerik.com/blazor-ui/components/sankey/elements/links#basic-customization) but the default color seems to be indeed set in HEX.

**Pull request merged**